cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -3254,6 +3285,26 @@ msgstr "Bydd eich sesiwn yn dod i ben os yw wedi bod yn anweithredol am hanner a
 msgid "Search"
 msgstr "Chwilio"
 
+#: cla_public/templates/checker/result/_find-legal-adviser.html:33
+msgid "Legal Aid is different in Scotland. Visit %(link)s for more information."
+msgstr "Mae Cymorth Cyfreithiol yn wahanol yn Yr Alban. Ewch i %(link)s i gael rhagor o wybodaeth."
```

Comment:
- Line 328 where it says
> Visit %(link)s for more information
and line 3290 where it says
> Ewch i %(link)s i gael...

Is the s required after the link interpolation? Just wanting it to be perfect :).

Reply:
The s after the link is required for the python syntax.  I actually omitted it at first and it took Said to explain why it wasn't working.